pkg/liqo-controller-manager/virtualNode-controller/virtualnode_controller.go

```diff
@@ -41,9 +41,16 @@ const (
 type VirtualNodeReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
+	// key = clusterID, value = technicalNamesapceName
+	TechnicalNamespacesNames map[string]string
 }
 
-// +kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch;create;update;patch;delete
+// cluster-role
+// +kubebuilder:rbac:groups=core,resources=nodes,verbs=get;watch;patch
+// +kubebuilder:rbac:groups=virtualKubelet.liqo.io,resources=namespacemaps,verbs=list;watch;delete;create
+// +kubebuilder:rbac:groups=discovery.liqo.io,resources=foreignclusters,verbs=get;list;watch
+
+// todo: servono privilegi apposta per lo stato ?
```

Comment:
This comment should be dropped. I think you do need dedicated permissions for subresources, what do you think @aleoli?

Reply:
Sorry, you need them if you need to update the status, if you only need reading permission I think that they are given by the parent resource (i.e. foreignclusters)

Reply:
thank you